content/statement/Skycoin Launches A Vision For The New Internet.pt.md

```diff
@@ -10,24 +10,24 @@ categories = [
 
 As principais características da Skycoin são velocidade, privacidade, segurança e o ecosistema.
 
-Em termos de velocidade a Skycoin compete com cartões de crédito e Apple pay. As transações ocorrem em segundos. Isso faz com que a moeda seja adequada para pontos de venda e transações da internet das coisas (IOT).
+Em termos de velocidade a Skycoin compete com cartões de crédito e Apple pay. As transações ocorrem em segundos. Isso faz com que a moeda seja adequada para pontos de venda (PoS) e transações da internet das coisas (IOT).
 
-A Skycoin atinge transações 100% privadas e anônimas graças ao uso do protocolo CoinJoin para embaralhar transações de uma mesma carteira. Ela também entrega um ambiente de transações seguras que utiliza criptografia para se proteger de ataques de 51% e outros tipos de ameaças comuns.
+A Skycoin atinge transações 100% privadas e anónimas graças ao uso do protocolo CoinJoin para combinar transações de mais de uma carteira. Ela também proporciona um ambiente de transações seguras que utiliza criptografia para se proteger de ataques de 51% e outros tipos de ameaças comuns.
 
-O ecosistema que engloba a Skycoin está se expandindo. Desenvolvedores estão criando aplicativos e ferramentas que incluem redes sociais, redes privadas (VPN) e mensageiros.
+O ecosistema que engloba a Skycoin está a expandir-se. Desenvolvedores estão criando aplicativos e ferramentas que incluem redes sociais, redes privadas (VPN) e mensageiros baseados na Skycoin.
 
-A Skycoin irá nivelar o mercado e criar um sistema de internet descentralizada distribuindo valor entre comunidades peer-to-peer. Ela entrega recursos públicos que eliminam a mineração e inflação que infestam outras criptomoedas.
+A Skycoin irá nivelar o mercado e criar um sistema de internet descentralizada distribuindo valor entre a comunidade peer-to-peer. Ela proporciona recursos públicos que eliminam a mineração e inflação que infestam outras criptomoedas.
 
-Mas os desenvolvedores dizem que o projeto Skycoin não é somente sobre criptomoeda. É também o combustível por trás da Skywire, uma rede peer-to-peer que corajosamente procura substituir a internet por algo melhor.
+Mas os desenvolvedores dizem que o projeto Skycoin não é somente sobre criptomoeda. É também o combustível por trás da Skywire, uma rede peer-to-peer que corajosamente procura substituir o cenário da internet atual por algo melhor.
 
-"Existem outros repositórios no [github.com/skycoin](https://github.com/skycoin) para redes em malha e protótipos de VPN distribuídas em que pessoas são pagas em tokens para encaminhar tráfego", diz Synth, o fundador do projeto Skycoin. "Existe também um serviço de distribuição de mídia social com replicação de dados peer-to-peer e diferentes outros projetos experimentais, assim como pesquisa em estruturas de dados imutáveis para a Internet da nova geração. Alguns deles são muito radicais."
```

Comment:
a frase sobre midia social está certa em portugues -BR 

sobre a pesquisa o meu está errado pq deveria ser no plural, mas investigação está incorreto ao meu ver.

Reply:
“There is also a distributed social media service, with peer-to-peer data replication and different experimental projects, as well as research into immutable data structures for next-generation internet. Some of them are very radical.”
research tem a ver com investigação tal como investigação em laboratório, não que se use propriamente um laboratório neste caso... é preciso dar a entender que não é andar à procura de algo que anda perdido.

Reply:
pesquisa em portugues - BR aparentemente significa o mesmo que investigação pra você